project3/styles.css

```
@@ -0,0 +1,373 @@
+    text-align: left;
+}
+
+.age {
+    margin-top: 38px;
+    margin-left: 16px;
+}
+
+.category {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.field-margin {
+    margin-top: 30px;
+    margin-bottom: 18px;
+    margin-left: 16px;
+}
+
+.margins {
+    margin-left: 18px;
+    margin-right: 18px;
+}
+
+#powers {
+    margin-left: 18px;
+    margin-right: 18px;
+    height: 68px;
+    width: 291px;
+}
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+
+.bulb {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+
+.muscle {
+    height: 101px;
+    width: 100px;
+}
+
+.watch {
+    height: 100px;
+    width: 100px;
+    margin-top: 40px;
+}
+
+.caption {
+    margin-top: 19px;
+    margin-left: 150px;
+    margin-right: 50px;
+}
+
+.text-box {
+    height: 73px;
+    width: 240px;
+    margin-left: 50px;
+    margin-right: 50px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+}
+
+svg {
+    display: block;
+    margin-right: 50px;
+    margin-left: 120px;
+    width: 100%;
+
+}
+
+
+
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #FFF;
+    --color-blue: #163346;
+
+}
+
+*,
+*::before *::after {
+    box-sizing: border-box;
+}
+
+
+
+h1 {
+    text-align: center;
+    font-family: 'Teko', sans-serif;
+    color: black;
+    font-size: 48px;
+    margin-top: 20px;
+    margin-bottom: 42px;
+}
+
+h2 {
+
+    margin-left: 16px;
+    font-family: 'BenchNine', sans-serif;
+
+}
+
+.great-power {
+    color: #163346;
+    margin-top: 42px;
+    margin-bottom: 57px;
+    font-size: 26px;
+    text-align: center;
+    font-family: 'BenchNine', sans-serif;
+}
+
+body {
+    background-color: var(--color-blue);
+    /* border: 10px solid var(--color-green); */
+    color: var(--color-offwhite);
+    /* font-family: var(--font-sans); */
+    font-family: 'BenchNine', sans-serif;
+    font-size: 22px;
+    position: relative;
+}
+
+
+
+.box-2 {
+    background-color: #1e2019;
+    margin-top: -20px;
+    margin-bottom: 47px;
+    margin-left: 24px;
+    margin-right: 24px;
+    position: absolute;
+    height: auto;
+    width: 327px;
+
+
+}
+
+
+
+.realname {
+    width: 100%;
+}
+
+.alias {
+    width: 100%;
+}
+
+.input-box {
+    margin-left: 16px;
+}
+
+.agebox {
+    height: 68px;
+    width: 109px;
+    margin-left: 16px;
+}
+
+.powers-and-abilities {
+    margin-left: 18px;
+    text-align: left;
+}
+
+.describe-powers {
+    height: 220px;
+    width: 291px;
+}
+
+.test {
+    width: 100%;
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: relative;
+    height: 1px;
+    width: 1px;
+}
+
+
+.button-reset {
+
+    /* background: transparent; */
+    background-color: #fff;
+    border: 10px solid #71f79f;
+    border-width: 10px;
+    cursor: pointer;
+    font-family: Teko;
```

Missing generic fallback font:
```suggestion
    font-family: Teko, sans-serif;
```